pkg/rpcclient/nep24/royalty.go

```diff
@@ -0,0 +1,163 @@
+/*
+Package nep24 contains RPC wrappers to work with NEP-24 contracts.
+
+Safe methods are encapsulated into RoyaltyReader structure.
```

Comment:
A couple of words about how to use this package in combination with `nep11` package.

Reply:
And BTW, this standard doesn't have unsafe methods.